File: tugas/AppPegawai/app/src/main/java/com/tugas/apppegawai/MainActivity.java
package com.tugas.apppegawai;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.RelativeLayout;

public class MainActivity extends AppCompatActivity {

    RelativeLayout btnGajiPegawai, btnPetunjuk, btnTentang, btnKeluar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Hooks
        btnGajiPegawai = findViewById(R.id.btnGajiPegawai);
        btnPetunjuk = findViewById(R.id.btnPetunjuk);
        btnTentang = findViewById(R.id.btnTentang);
        btnKeluar = findViewById(R.id.btnKeluar);

        btnGajiPegawai.setOnClickListener(v -> {
            Intent intent = new Intent(this, GajiPegawai.class);
            startActivity(intent);
        });

        btnPetunjuk.setOnClickListener(v -> {
            Intent intent = new Intent(this, Petunjuk.class);
            startActivity(intent);
        });

        btnTentang.setOnClickListener(v -> {
            Intent intent = new Intent(this, Tentang.class);
            startActivity(intent);
        });

        btnKeluar.setOnClickListener(v -> {
            Intent intent = new Intent(this, Keluar.class);
            startActivity(intent);
        });
    }
}

File: tugas/tugas1/app/src/main/java/com/example/tugas1/MainActivity.kt
package com.example.tugas1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val editTextNilaiA = findViewById<EditText>(R.id.editTextNilaiA)
        val editTextNilaiB = findViewById<EditText>(R.id.editTextNilaiB)
        val btnPlus = findViewById<Button>(R.id.btnPlus)
        val btnMin = findViewById<Button>(R.id.btnMin)
        val btnTime = findViewById<Button>(R.id.btnTime)
        val btnDivide = findViewById<Button>(R.id.btnDivide)
        val btnClear = findViewById<Button>(R.id.btnClear)
        val textViewHasil = findViewById<TextView>(R.id.textViewHasil)

        btnPlus.setOnClickListener() {
            val nilaiA = editTextNilaiA.text.toString().toInt()
            val nilaiB = editTextNilaiB.text.toString().toInt()
            val hasil = nilaiA + nilaiB
            textViewHasil.text = hasil.toString()
        }

        btnMin.setOnClickListener() {
            val nilaiA = editTextNilaiA.text.toString().toInt()
            val nilaiB = editTextNilaiB.text.toString().toInt()
            val hasil = nilaiA - nilaiB
            textViewHasil.text = hasil.toString()
        }

        btnTime.setOnClickListener() {
            val nilaiA = editTextNilaiA.text.toString().toInt()
            val nilaiB = editTextNilaiB.text.toString().toInt()
            val hasil = nilaiA * nilaiB
            textViewHasil.text = hasil.toString()
        }

        btnDivide.setOnClickListener() {
            val nilaiA = editTextNilaiA.text.toString().toInt()
            val nilaiB = editTextNilaiB.text.toString().toInt()
            val hasil = nilaiA / nilaiB
            textViewHasil.text = hasil.toString()
        }

        btnClear.setOnClickListener() {
            editTextNilaiA.text = null
            editTextNilaiB.text = null
            textViewHasil.text = null
        }
    }
}